tests/e2e/flowtest/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import os.path
+
+import pytest
+import yaml
+from ocs_ci.ocs.resources import pod
+import time
+from tempfile import mkdtemp
+import os
+from ocs_ci.ocs.resources.pod import validate_pods_are_respinned_and_running_state
+from ocs_ci.framework.testlib import E2ETest, ignore_leftovers
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.helpers import helpers
+import urllib.request
+from ocs_ci.ocs.resources.pod import delete_deploymentconfig_pods
+from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+
+
+TARFILE = "file.gz"
+ntar_loc = mkdtemp()
+
+log = logging.getLogger(__name__)
+
+
+class TestSelinuxrelabel(E2ETest):
+    @pytest.fixture()
+    def download_files(self):
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.5.tar.gz"
+        download_path = ntar_loc
+        dir_path = os.path.join(os.getcwd(), download_path)
+        print(dir_path)
+        file_path = os.path.join(dir_path, "file.gz")
+        print(file_path)
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+        print(dir_path)
+        return dir_path
+
+    @pytest.fixture()
+    def create_pvc_and_deploymentconfig_pod(self, request, pvc_factory):
+        """"""
+
+        def finalizer():
+            delete_deploymentconfig_pods(pod_obj)
+
+        request.addfinalizer(finalizer)
+
+        # Create pvc
+        pvc_obj = pvc_factory(size=20)
+
+        # Create service_account to get privilege for deployment pods
+        sa_name = helpers.create_serviceaccount(pvc_obj.project.namespace)
+
+        helpers.add_scc_policy(
+            sa_name=sa_name.name, namespace=pvc_obj.project.namespace
+        )
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.project.namespace,
+            sa_name=sa_name.name,
+            dc_deployment=True,
+        )
+        helpers.wait_for_resource_state(
+            resource=pod_obj, state=constants.STATUS_RUNNING
+        )
+        return pod_obj, pvc_obj
+
+    def copy_files(self, pod_obj, pvc_obj):
+        log.info(f"pod obj name---- {pod_obj.name}")
+        pod_name = pod_obj.name
+
+        log.info("cephfs pod created")
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_obj.project.namespace,
+        )
+        # Number of times we copy the kernel
+        copies = 3
```

Comment:
instead of hardcoding this way, I feel it is better to pass it as an arg to provide more flexibilty. JIC, if we want to increase the file count we could simply increase the count